### PR TITLE
Allow clearing of auto archive

### PR DESCRIPTION
**Pull request by petros1999**
As new contributors, we would appreciate any suggestions or guidelines in order to make this pr succesfull. Thanks in advance

### CODE REVIEW

**Comment by petros1999** [comment]
This pull request refers to issue #3944 
We have added a clear() method in FileArchive class of io.py and the corresponding test case.
Also we have added an if statement in auto() method of NotebookArchive class in which we call the clear() method.

File: holoviews/tests/core/testarchives.py [hunk 1]
```diff
@@ -124,3 +124,14 @@ def test_filearchive_json_single_file(self):
...
+    """
+    A test case that examines the clear() method of
+    FileArchives of io.py
+    """
+    def test_filearchive_clear_file(self):
+        export_name = "archive_for_clear"
+        archive = FileArchive(export_name=export_name, pack=False)
+        archive.add(self.image1)
+        archive.add(self.image2)
+        self.assertEqual(None, FileArchive.clear(archive))
```
Comment (philippjfr) [inline holoviews/tests/core/testarchives.py:137]: Not sure what this is testing, shouldn't this be something like:

```suggestion
        archive.clear()
        self.assertEqual(archive._files, {})
```

File: holoviews/core/io.py [hunk 2]
```diff
@@ -854,3 +854,10 @@ def contents(self, maxlen=70):
     def listing(self):
         "Return a list of filename entries currently in the archive"
         return ['.'.join([f,ext]) if ext else f for (f,ext) in self._files.keys()]
+
+    def clear(self):
+        "Clears the file archive"
+        return self._files.clear()
```
Comment (philippjfr) [inline holoviews/core/io.py:860]: Don't think `dict.clear()` returns anything

```suggestion
        self._files.clear()
```

**Comment by philippjfr** [comment]
Much appreciated, thanks for contributing!

**Pull request merged**